src/Executor/Client.php

```diff
@@ -0,0 +1,335 @@
+<?php
+
+namespace OpenRuntimes\Executor;
```

Comment:
Theoretically, this is the PHP SDK. Do we need it here or can we keep in Appwrite until we actually have an SDK for this server?

Reply:
Client.php was removed. We now do simple CURL request in place where it was used.